changelog/undistributed/changelog_fix_show_interfaces_switchport_20210315102854.rst

```diff
@@ -0,0 +1,3 @@
+* IOSXE
+  * Modified show_interface.py to fix a bug in ShowInterfacesSwitchport
+  * Also modified ios/iosxe current tests because they were wrong
```

Comment:
Please add change log header and fix indentations
--------------------------------------------------------------------------------
                                Fix
--------------------------------------------------------------------------------
* IOS
    * Modified <class name>:
        * Modified show_interface.py to fix a bug in ShowInterfacesSwitchport
        * Modified ios current tests because they were wrong

* IOSXE
    * Modified <class name>:
        * Modified show_interface.py to fix a bug in ShowInterfacesSwitchport
        * Modified iosxe current tests because they were wrong

Reply:
done and commited